connect/src/com/telenor/connect/utils/ConnectUtils.java

```diff
@@ -49,4 +53,11 @@ public static void sendTokenStateChanged(boolean state) {
         intent.putExtra(LOGIN_STATE, state);
         localBroadcastManager.sendBroadcast(intent);
     }
+
+    private static Map<String, String> getErrorParams(String error, String errorDescription) {
```

Comment:
I'm not sure this abstraction is worth it. It's one indirection more, and getErrorParams is used in one place only anyway.